frontend/packages/pipelines-plugin/src/components/pipelines/modals/triggers/submit-utils.ts

```diff
@@ -43,7 +43,7 @@ export const exposeRoute = async (elName: string, ns: string, iteration = 0) =>
 
     // Get the service, find out what port we are exposed on
     const serviceResource = await k8sGet(ServiceModel, serviceGeneratedName, ns);
-    const servicePort = serviceResource.spec?.ports?.[0]?.targetPort;
+    const servicePort = serviceResource.spec?.ports?.[0]?.name;
```

Comment:
Just some questions and I really don't know the answer :smirk: 

1. How does this work together with different Pipeline operator versions?
2. Should we maybe fallback to `name` if `targetPort` is not defined? (...targetPort || ...name?) or are we sure that this is now always the name?
3. Why is no test update needed? :smirk: :smile:

Reply:
Here is the doc for the root cause and the solution suggested by the Pipeline team https://docs.google.com/document/d/16VCW6LOzuuaJYvtrS707T6R9dMyp-PoyXNgquCRYHFA/edit?pli=1. It will work with all the pipeline versions and should always be `name`.